web/package.json

```diff
@@ -104,6 +104,7 @@
   "homepage": "https://github.com/keymanapp/keyman#readme",
   "devDependencies": {
     "@keymanapp/resources-gosh": "*",
+    "@playwright/test": "^1.46.1",
```

Comment:
We already use other parts of playwright in the test runner. This adds the test component that allows us to start and manipulate a browser from within a test - the test runner is the reverse: it runs the tests in the browser. For e2e tests however IMO it is better to run the tests outside of the browser and simulate the behavior of a user.